src/runtime/pay-flow.ts

```diff
@@ -123,13 +123,13 @@ export class PayStartFlow {
   async start(): Promise<void> {
     // Get the paySwgVersion for buyflow.
     const clientConfig = await this.clientConfigManager_.getClientConfig();
-    this.start_(clientConfig.paySwgVersion);
+    this.start_(!clientConfig.useUpdatedOfferFlows, clientConfig.paySwgVersion);
   }
 
   /**
    * Starts the payments flow for the given version.
    */
-  private start_(paySwgVersion?: string): void {
+  private start_(useClassicFlow: boolean, paySwgVersion?: string): void {
```

Comment:
Would it be clearer if we didn't have to negate the input parameter, and instead this was `usePayFlow`?

Reply:
ok